app/models/spree/shipment.rb

```diff
@@ -168,6 +168,15 @@ def editable_by?(_user)
       !shipped?
     end
 
+    def update_amounts
+      return unless selected_shipping_rate
+
+      self.update_columns(
```

Comment:
I guess that using `update_columns` is a sign that we have too many callbacks. Out of scope here. :wink: